source/fundamentals/logging.txt

```diff
@@ -28,6 +28,7 @@ You can configure the driver to report logger messages at three different
 levels as shown in the following table.
 
 .. list-table::
+   :widths: 20 80
```

Comment:
I wonder if the first column could be a slightly higher percentage. When the window is slightly too narrow to fit the full width of every part of the page, there's some wrapping in the Class name (first) column which is worse than wrapping in the Description (second) column.
In the monitoring page, you had a little more space in the first column which padded the width such that it didn't immediately wrap when the browser window was not wide enough.